fairseq/tasks/sentence_prediction.py

```diff
@@ -223,7 +223,7 @@ def source_dictionary(self):
 
     @property
     def target_dictionary(self):
-        return self.dictionary
+        return self._label_dictionary
```

Comment:
Can you please keep it `self.dictionary` and instead pass `self.label_dictionary` on the `line 167`?
With your current changes, loading of encoder-decoder based finetuning (`BART`) will break.

Reply:
Sure thing, there it is:)